martin/src/config/file/main.rs

```diff
@@ -218,15 +233,21 @@ impl Config {
         #[cfg(feature = "pmtiles")]
         let pmtiles_cache = cache_config.create_pmtiles_cache();
 
+        #[cfg(feature = "_tiles")]
+        let (tiles, warnings) = self
+            .resolve_tile_sources(
+                &resolver,
+                #[cfg(feature = "pmtiles")]
+                pmtiles_cache,
+            )
+            .await?;
+
+        #[cfg(feature = "_tiles")]
+        OnInvalid::handle_warnings(self.on_invalid, &warnings)?;
```

Comment:
to match below + above

```suggestion
        self.on_invalid.unwrap_or_default().handle_tile_warnings(&warnings)?;
```